Documentation/contributing/development/contributing_guide.rst

```diff
@@ -14,29 +14,31 @@ Clone and Provision Environment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 #. Make sure you have a `GitHub account <https://github.com/join>`_
-#. Clone the cilium repository into your ``GOPATH``.
+#. Fork the Cilium repository to your GitHub user or organization.
+#. Clone your ``${YOUR_GITHUB_USERNAME_OR_ORG}/cilium`` fork into your ``GOPATH``, and setup the base repository as ``upstream`` remote:
 
    ::
 
-      mkdir -p $GOPATH/src/github.com/cilium
-      cd $GOPATH/src/github.com/cilium
-      git clone https://github.com/cilium/cilium.git
+      mkdir -p ${GOPATH}/src/github.com/cilium
+      cd ${GOPATH}/src/github.com/cilium
```

Comment:
Nit: Double quotes? In case `GOPATH` contains weird things like _spaces_?

Reply:
Fixed!